modules/ui/src/main/scala/lucuma/ui/aladin/ReactAladin.scala

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.aladin
+
+import cats.syntax.all.*
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.hooks.*
+import japgolly.scalajs.react.hooks.Hooks.UseState
+import japgolly.scalajs.react.vdom.TopNode
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.math.*
+import lucuma.react.common.*
+import lucuma.ui.aladin.facade.*
+import org.scalajs.dom.html
+
+import scala.scalajs.js
+
+type Aladin = lucuma.ui.aladin.facade.JsAladin
+
+extension (a: Aladin)
```

Comment:
Why not put these in `object JsAladin` ?

Reply:
I wanted `JsAladin` to be a pure JS facade. Instead these methods are mostly to make it more react friendly